socketdock/api.py

```diff
@@ -52,6 +53,34 @@ async def socket_send(request: Request, connectionid: str):
     await socket.send(request.body)
     return text("OK")
 
+@api.post("/socket/<connectionid>/send_text")
+async def socket_send(request: Request, connectionid: str):
+    """Send a message to a connected socket."""
+    if LOGGER.isEnabledFor(logging.INFO):
+        LOGGER.info("Inbound message for %s", connectionid)
+        LOGGER.info("Existing connections: %s", active_connections.keys())
+
+    if connectionid not in active_connections:
+        return text("FAIL", status=500)
+
+    socket = active_connections[connectionid]
+    await socket.send(request.json["text"])
```

Comment:
It seems like this would be better controlled by handling the request mime type rather than having a separate endpoint with a specific payload. Perhaps if the mime type is `text/plain` we can send the value to the websocket as `TEXT` (by decoding `request.body`) and then for other mime types, we can send as `BINARY` as before (by not decoding `request.body`).

Reply:
It seems that sanic framework doesn't use the content-type to detect the incoming content by default with the exception of json. I tried passing in a different content-type to sanic, but it still processes the body type as binary instead of text. I figure by making a second call we could be explicit about the expected behavior.

Reply:
`await socket.send(request.body.decode())` will send `TEXT`. `await socket.send(request.body)` will send `BINARY`.

Reply:
See https://github.com/hyperledger/aries-socketdock/pull/10; we actually explicitly changed it to send `BINARY` recently as this is the expected message type for ACA-Py